extensions/amp-ad/amp-ad.md

```diff
@@ -31,7 +31,7 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Required Script</strong></td>
-    <td><code>&lt;script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js">&lt;/script></code> Note: amp-ad may still work without this script, but we highly recommend it for future compatibility</td>
+    <td><code>&lt;script async custom-element="amp-ad" src="https:&sol;&sol;cdn.ampproject.org&sol;v0&sol;amp-ad-0.1.js">&lt;/script></code> Note: amp-ad may still work without this script, but we highly recommend it for future compatibility</td>
```

Comment:
Thanks for your review - apologies for having you do that many comments - so my turn again ;) 
Just a sidenote on this change: i could not get CI through without that change. CI saw them as dead links; will revert nonetheless.